### DISCUSSION TITLE
maximum number of supported databases for docker

### DISCUSSION THREAD

**Discussion by adammakowskidev** [dragonflydb/dragonfly#729] [Q&A]
Hi
I use Dragonfly using the docker image. I would like to run it in such a way that I have for example 100 databases? Is this possible?
below is the command in which I started Dragonfly, how to modify it?

`docker run --restart unless-stopped --network=enhance-network --ip=199.99.88.255 --name dragonfly --ulimit memlock=-1 -d docker.dragonflydb.io/dragonflydb/dragonfly
`

I try use --dbnum 100 but the command doesn't work and docker won't start. Any ideas?
Thx!

**Comment by romange** [comment]
`docker run  --network=host   --ulimit memlock=-1 docker.dragonflydb.io/dragonflydb/dragonfly  --dbnum=100` works for me
